util/encodings.go

```diff
@@ -115,3 +117,20 @@ var emptyASN1Sequence = []byte{0x30, 0x00}
 func IsEmptyASN1Sequence(input []byte) bool {
 	return len(input) < 2 || bytes.Equal(input, emptyASN1Sequence)
 }
+
+// ParseBMPString returns a uint16 encoded string following the specification for a BMPString type
+func ParseBMPString(bmpString []byte) (string, error) {
+	if len(bmpString)%2 != 0 {
+		return "", errors.New("pkcs12: odd-length BMP string")
```

Comment:
pkcs12?

Reply:
Removed, sorry. That was because I pulled some functionality from a pending golang PR.